sys/include/net/credman.h

```diff
@@ -90,9 +90,10 @@ typedef uint16_t credman_tag_t;
  * @brief Credential types
  */
 typedef enum {
-    CREDMAN_TYPE_EMPTY  = 0,
-    CREDMAN_TYPE_PSK    = 1,
-    CREDMAN_TYPE_ECDSA  = 2,
+    CREDMAN_TYPE_EMPTY  = 0,    /**< Empty type. Used to detect uninitialized
```

Comment:
```suggestion
    /**
     * @brief Empty type
     *
     * Used to detect uninitialized values internally. 
     */
    CREDMAN_TYPE_EMPTY  = 0,
```

Reply:
Or rather "objects" instead of "values"

Reply:
How about `[...] uninitialized @ref credman_credential_t objects internally.`?

Reply:
:+1: